theano/tensor/tests/test_opt.py

```diff
@@ -137,7 +138,8 @@ def test_elim3(self):
         dimshuffle_lift.optimize(g)
         self.assertTrue(str(g) == "[x]", str(g))
         # Check stacktrace was copied over correctly after opt was applied
-        self.assertTrue(check_stack_trace(g, ops_to_check='all'))
+        self.assertTrue(check_stack_trace(g, ops_to_check='all',
+                                          bug_print='ignore'))
```

Comment:
Also here, `'all'` will not work with the correct version of `check_stack_trace`, so you would have to either pass `ops_to_check=DimShuffle`, or remove the check and add a comment.